playground/index.php

```diff
@@ -1,4 +1,8 @@
 <?php
 
+use DDTrace\Tracer;
+
 error_log("Printed to the error log");
 echo "This is the playground!\n";
+
+echo "Tracer version: " . Tracer::version() . "\n";
```

Comment:
Should we wrap this in a `class_exists` check in case we want to test without a request_init_hook or disabled ddtrace or something?

```suggestion
if (class_exists('DDTrace\Tracer', $autoload = false)) {
    echo "Tracer version: " . Tracer::version() . "\n";
}
```

Reply:
TIL about `$autoload = false`